spec/minimax_spec.rb

```diff
@@ -0,0 +1,98 @@
+require './helpers/Helper'
+require '../ttt/Minimax'
+require '../ttt/Board'
+require '../ttt/Ai'
+require '../ttt/Symbols'
+
+describe Minimax do
+  before(:each) do
+    @board = Board.new(3)
+    @player = Ai.new
+    @symbols = Symbols.new('x', 'o')
+    @helper = Helper.new
+  end
```

Comment:
An alternative here is to use `let`, which creates a variable and eager loads a block you defined.

```rb
let(:helper) { Helper.new }
```

You can then reference `helper` in your tests below.